pkg/kernel/neighbor/monitor.go

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I guess it's ok, but I would still prefer to have metrics for with the IP / MAC as attribute and the state as value than having the neighbor table added to the logs.

Reply:
Proper metrics for Targets would be nice indeed.
(Though currently I think MAC might change even without unregistering the target first. Also, MAC is not known until successfully resolved, meaning the key won't be complete from start.)